app/server/utils/user.py

```diff
@@ -283,6 +282,55 @@ def create_transfer_account_user(first_name=None, last_name=None, preferred_lang
     return user
 
 
+def create_user_without_transfer_account(phone):
+    """
+    This method creates a user without a transfer account or blockchain wallet.
+    :param phone: string with user's msisdn
+    :return: User
+    """
+    user = User(phone=phone, signup_method=SignupMethodEnum.USSD_SELF_SIGNUP.value)
```

Comment:
When trying to signup with the AfricasTalking sandbox, I get the following error generated from this line:  
`TypeError: 'signup_method' is an invalid keyword argument for User`

Reply:
Is the attribute in your user table. 
I had to manually resolve some migration error there because the ```python manage.py db merge heads```  wasn't working.

Reply:
To be precise it was generating a version that didn't exactly resolve my issue.

Reply:
`sign_up_method` is in my table, and the migrations ran without a problem. One thing I did notice after reading your comment and checking the database, is that this line says `signup_method`, instead of `sign_up_method`. I _can_ sign up with that changed (pushed that change as well), but the other bugs remain.

Reply:
I think you should add a couple more tests, since they didn't detect this error at all either.